pkg/pool-weighted/contracts/smart/ManagedPool.sol

```diff
@@ -349,6 +356,35 @@ contract ManagedPool is BaseWeightedPool, ReentrancyGuard {
         emit SwapEnabledSet(swapEnabled);
     }
 
+    /**
+     * @dev Getter for the sum of all weights. In initially FixedPoint.ONE, it can be higher or lower
+     * as a result of adds and removes.
+     */
+    function getTotalWeight() external view returns (uint256) {
+        return _totalWeight;
+    }
+
+    /**
+     * @dev Placeholder function for testing. Adding or removing a token may cause the total weight to
+     * diverge from ONE.
+     */
+    function _setTotalWeight(uint256[] memory normalizedWeights, uint256 totalWeight) internal {
+        InputHelpers.ensureInputLengthMatch(_getTotalTokens(), normalizedWeights.length);
+        _require(totalWeight > WeightedMath._MIN_WEIGHT, Errors.MIN_WEIGHT);
+
+        if (totalWeight == _totalWeight) {
+            return;
+        }
+
+        _totalWeight = totalWeight;
```

Comment:
It seems like this achieves nothing. What's the point of the `totalWeight` param?

Reply:
It's a placeholder for testing. This won't be in the final code. It's a way to simulating adding and removing, which would affect the sum, by just directly setting it.